lifemonitor/api/models/rocrate.py

```diff
@@ -100,7 +100,8 @@ def dataset_name(self):
 
     @property
     def main_entity_name(self):
-        return self._roc_helper.mainEntity['name']
+        mainEntity = self._roc_helper.mainEntity
+        return mainEntity['name'] if mainEntity and 'name' in mainEntity else None
```

Comment:
Defining "name" for the main workflow is not required in Workflow RO-Crate, so we should try to accept the crate in this case. We can fall back to the main entity's id, which is always defined:

```python
return mainEntity.get("name", mainEntity.id) if mainEntity else None
```